src/content/en/2019/performance.md

```diff
@@ -30,232 +30,177 @@ Almost all of the other chapters in the Web Almanac are based on data from the [
-
-The distribution of fast, average, and slow websites appears to be more balanced, with 26.61% of websites qualifying as fast and 31.35% as slow.
+Compared to the previous metrics, the distribution of aggregate FID performance is much more skewed towards fast and moderate experiences than slow. 40% of websites have fast FID and only 15% have slow FID. The nature of FID being an interactivity metric -- as opposed to a loading metric bound by network speeds -- makes for an entirely different way to characterize performance.
 
 #### FID by device
 
-<figure>
-// Chart: Flame distribution of 07_02b
-<figcaption>Figure 19. Distribution of _desktop_ websites' fast, average, and slow FID performance.</figcaption>
+<figure id="fig14">
+<iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=16379508&amp;format=interactive"></iframe>
+<figcaption>Figure 14. Distribution of <em>desktop</em> websites' fast, moderate, and slow FID performance.</figcaption>
 </figure>
 
-<figure>
-// Chart: Flame distribution of 07_02c
-<figcaption>Figure 20. Distribution of _phone_ websites' fast, average, and slow FID performance.</figcaption>
+<figure id="fig15">
+<iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=519511409&amp;format=interactive"></iframe>
+<figcaption>Figure 15. Distribution of <em>phone</em> websites' fast, moderate, and slow FID performance.</figcaption>
 </figure>
 
 Breaking FID down by device, it becomes clear that there are two very different stories. Desktop users enjoy fast FID almost all the time. Sure there are some websites that throw out a slow experience now and then, but the results are predominantly fast. Mobile users, on the other hand, have what seem to be one of two experiences: pretty fast (but not quite as often as desktop) and almost never fast. The latter is experienced by users on only the tail ~10% of websites, but this is still a substantial difference.
 
-<figure>
-// Chart: Bar distributions of 07_04b
-<figcaption>Figure 21. Distribution of websites labelled as having fast, average, or slow FID, broken down by device type.</figcaption>
-</figure>
-
-<figure>
-
-Device | Fast FID | Average FID | Slow FID
--- | -- | -- | --
-desktop | 70.32% | 23.20% | 6.48%
-phone | 13.76% | 43.21% | 43.03%
-
-<figcaption>Figure 22. Table of websites labelled as having fast, average, or slow FID, broken down by device type.</figcaption>
+<figure id="fig16">
+<iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSQlf3_ySLPB5322aTumUZhbVGdaUdkmi1Hs4bYuO3Z1kqM4xspx7REbwXukwPd_tsOSg6oImzpYLM9/pubchart?oid=1533541692&amp;format=interactive"></iframe>
```

Comment:
Are all these embed links supposed to have `&amp;`?

Reply:
Yes, this comes from Sheets when publishing a chart

![image](https://user-images.githubusercontent.com/1120896/67834667-6367fb80-fabe-11e9-8599-7c05ad826b6a.png)